backend/src/dao/controller/project.controller.ts

```diff
@@ -42,15 +40,19 @@ export class ProjectController {
   @Put(':projectId')
   @HttpCode(200)
   @Auth(Role.projectManager, Role.founder)
-  update(@Param('projectId') projectId: string, @Body() projectDto: ProjectDto): Promise<ProjectDto> {
+  update(
+    @Param('daoId') daoId: string,
+    @Param('projectId') projectId: string,
+    @Body() projectDto: ProjectDto,
+  ): Promise<ProjectDto> {
     projectDto.projectId = projectId
     return this.projectService.update(projectDto)
   }
 
   @Delete(':projectId')
   @HttpCode(200)
   @Auth(Role.projectManager, Role.founder)
-  delete(@Param('projectId') projectId: string): Promise<void> {
+  delete(@Param('daoId') daoId: string, @Param('projectId') projectId: string): Promise<void> {
```

Comment:
How about methods to find all contributors currently participating (after invitation) in a project? We will probably need something in the UI to show project board with task status and contributors assigned to tasks.

Reply:
Good suggestion, I will add this to #58 for founders/PMs